persons.txt

```diff
@@ -0,0 +1 @@
+[{"name":"Andrew","height":"88","hair":"black","happiness":"10"},{"name":"Marta","height":"88","hair":"black"},{"name":"Marta","height":"88","hair":"black"}]
```

Comment:
Do you need to store this file? Maybe add it to gitignore as this is data or the running application?

Reply:
done!